modules/ebi-metagenomics/jbrowse/sortgff/main.nf

```diff
@@ -0,0 +1,40 @@
+process SORT_GFF {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container 'quay.io/biocontainers/coreutils:8.25--0'
```

Comment:
This needs to consider the singularity image too, like this example:
```
    conda "bioconda::blast=2.14.1"
    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
        'https://depot.galaxyproject.org/singularity/blast:2.14.1--pl5321h6f7f691_0':
        'biocontainers/blast:2.14.1--pl5321h6f7f691_0' }"
```
Was this module created with the nf-core tools?